db_windows.go

```diff
@@ -21,8 +21,7 @@ import (
 
 func mmap(f *os.File, sz int) ([]byte, error) {
 	low, high := uint32(sz), uint32(sz>>32)
-
-	h, errno := syscall.CreateFileMapping(syscall.Handle(f.Fd()), nil, syscall.PAGE_READONLY, low, high, nil)
+	h, errno := syscall.CreateFileMapping(syscall.Handle(f.Fd()), nil, syscall.PAGE_READONLY, high, low, nil)
```

Comment:
That's a pretty significant change, did I get the order wrong?
Seems so from https://msdn.microsoft.com/en-us/library/windows/desktop/aa366537(v=vs.85).aspx I suppose.

Reply:
yes, that make it create a huge mmap. so  CreateFileMapping failed.